### ISSUE TITLE
Pattern: Lead Space - Accessibility issue for primary and secondary buttons

### ISSUE THREAD

**Issue by chetankan** [carbon-design-system/ibm-dotcom-library#1229]
## Detailed description
Issue:-
Tab order is not logical.
1. First focus is going to primary action button and second focus is going to secondary action button , but visually first secondary button is present , second primary action button is present.
Note:visual order and DOM order is not matching"
2. JAWS (Screen reader) - when the focus comes to the mentioned controls screen reader is reading Primary action button button and secondary action button button.

Recommended fix:- 
1. Tab order should be logical. Visual order DOM order should match."
2. The role is already conveyed  the text "button" can be removed from the label.

## Additional information

![Screenshot.png](https://images.zenhubusercontent.com/5e21a841639f517d456f7e43/a1885486-3624-4b48-a438-52333d76cdc5)
